### PR TITLE
Migrate Home page to new macaw

### PR DESCRIPTION
**Pull request by poulch**
closes https://github.com/saleor/saleor-dashboard/issues/3734

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `data-test-id` are added for new elements
4. [ ] The changes are tested in Chrome/Firefox/Safari browsers and in light/dark mode
5. [ ] Your code works with the latest stable version of the core

### Test environment config

API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
APPS_MARKETPLACE_API_URI=https://apps.staging.saleor.io/api/v2/saleor-apps

### Do you want to run more stable tests?

To run all tests, just select the stable checkbox. To speed up tests, increase the number of containers. Tests will be re-run only when the "run e2e" label is added.

1. [x] stable
2. [ ] giftCard
3. [ ] category
4. [ ] collection
5. [ ] attribute
6. [ ] productType
7. [ ] shipping
8. [ ] customer
9. [ ] permissions
10. [ ] menuNavigation
11. [ ] pages
12. [ ] sales
13. [ ] vouchers
14. [x] homePage
15. [ ] login
16. [ ] orders
17. [ ] products
18. [ ] app
19. [ ] plugins
20. [ ] translations
21. [ ] navigation
22. [ ] variants
23. [ ] payments

CONTAINERS=5

### CODE REVIEW

File: src/home/components/HomeNotificationList/HomeNotificationList.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    }
+
+    if (productsOutOfStock === 0) {
+      return intl.formatMessage(messages.noProductsOut);
+    }
+
+    return intl.formatMessage(messages.productOut, {
+      amount: <strong>{productsOutOfStock}</strong>,
+    });
+  };
```
Comment (Droniu) [inline src/home/components/HomeNotificationList/HomeNotificationList.tsx:74]: **suggestion**: these functions should be outside this component

File: src/home/components/HomeProductList/HomeProductList.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  testId,
+}: HomeProductListProps) => {
+  const intl = useIntl();
+
+  return (
+    <Box data-test-id={testId}>
+      <Text variant="heading" display="block" paddingTop={7} marginBottom={2}>
+        {intl.formatMessage({
+          id: "rr8fyf",
+          defaultMessage: "Top Products",
```
Comment (Droniu) [inline src/home/components/HomeProductList/HomeProductList.tsx:29]:
```suggestion
          defaultMessage: "Top products",
```

File: src/home/components/HomeProductList/HomeProductList.tsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+                      <Text size="small" color="textNeutralSubdued">
+                        {variant.attributes
+                          .map(attribute => attribute.values[0].name)
+                          .join(" / ")}
+                      </Text>
+
+                      <Text size="small" color="textNeutralSubdued">
+                        <FormattedMessage
+                          id="0opVvi"
+                          defaultMessage="{amount, plural,one {One ordered}other {{amount} Ordered}}"
```
Comment (Droniu) [inline src/home/components/HomeProductList/HomeProductList.tsx:70]:
```suggestion
                          defaultMessage="{amount, plural,one {One ordered}other {{amount} ordered}}"
```

File: src/hooks/useMediaQuery.ts [hunk 4]
```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from "react";
+
+// Source: https://github.com/juliencrn/usehooks-ts/blob/master/packages/usehooks-ts/src/useMediaQuery/useMediaQuery.ts
+export function useMediaQuery(query: string): boolean {
```
Comment (Droniu) [inline src/hooks/useMediaQuery.ts:4]: **suggestion**: can't we just install this lib? It's not large and hooks are pretty useful

**Review by Droniu — APPROVED** [review]

File: src/home/components/HomeActivityCard/HomeActivityCard.tsx [hunk 5]
```diff
@@ -1,104 +1,86 @@
...
-              <ListItem key={activityId} className={classes.listItem}>
+              <List.Item
+                key={activityId}
+                flexDirection="column"
+                alignItems="flex-start"
+                cursor="auto"
+                paddingY={1}
+                paddingX={6}
+                __marginLeft={"-" + themeValues.spacing[6]}
+                __marginRight={"-" + themeValues.spacing[6]}
```
Comment (Droniu) [inline src/home/components/HomeActivityCard/HomeActivityCard.tsx:48]: **question**: that looks weird, are negative margins absolutely necessary here?

File: src/home/components/HomeNotificationList/utils.tsx [hunk 6]
```diff
@@ -0,0 +1,56 @@
+import Skeleton from "@dashboard/components/Skeleton";
+import React from "react";
+import { IntlShape } from "react-intl";
+
+import { homeNotificationTableMessages as messages } from "./messages";
+
+export const getOrderTpFullfillText = (
```
Comment (Droniu) [inline src/home/components/HomeNotificationList/utils.tsx:7]:
```suggestion
export const getOrderToFullfillText = (
```

**Review by Droniu — DISMISSED** [review]

**Review by andrzejewsky — CHANGES_REQUESTED** [review]

File: src/home/components/HomeActivityCard/HomeActivityCard.tsx (hunk 5 above)
Comment (poulch) [reply src/home/components/HomeActivityCard/HomeActivityCard.tsx:48]: yes

**Review by andrzejewsky — DISMISSED** [review]

File: src/index.css [hunk 7]
```diff
@@ -48,3 +48,9 @@ body {
 [id*="ScrollableDialog"] .infinite-scroll-component {
   overflow: hidden !important;
 }
+
+@media screen and (max-width: 1024px) {
+  .mobile-full-height {
+    height: auto !important;
+  }
+}
```
Comment (Droniu) [inline src/index.css:56]: **suggestion**: can we also reference https://github.com/saleor/macaw-ui/issues/498 here?

File: src/home/components/HomeNotificationList/HomeNotificationListItem.tsx [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+  >
+    <Link className={sprinkles({ width: "100%" })} to={linkUrl}>
+      <Box
+        display="flex"
+        justifyContent="space-between"
+        alignItems="center"
+        width="100%"
+      >
+        <Text size="small">{children}</Text>
+        <KeyboardArrowRight />
```
Comment (Droniu) [inline src/home/components/HomeNotificationList/HomeNotificationListItem.tsx:33]: **question**: This is a new MUI import, do we need it? I think ChevronRight from macaw is suitable here

**Review by Droniu — DISMISSED** [review]

**Review by Droniu — APPROVED** [review]

**Review by andrzejewsky — DISMISSED** [review]

**Review by Droniu — DISMISSED** [review]

**Comment by wojteknowacki** [comment]
Not the whole element is clickable on the Home screen

https://github.com/saleor/saleor-dashboard/assets/124166231/ec98754c-b731-42be-a0e3-28d46ee4a9c8


https://github.com/saleor/saleor-dashboard/assets/124166231/52ef6f00-b0b4-48fe-b5b7-7cf49612b6c8

**Review by Droniu — APPROVED** [review]

**Pull request merged**